CHANGELOG

```diff
@@ -2,6 +2,35 @@
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.2] - 2022-12-15
+### Added
+- help-verbose subcommand to print out all flags #1171 #1180
+- Support for Redis cluster #1181
+- Allow loopback candidates to be used via config option #1185
+- Support for high bitrate audio #1188
+- Ability to detect publication errors and force reconnect #1214
+- API secrets are validated upon startup to ensure sufficient security #1217
+
+### Fixed
+- Correctly suppress verbose pion logs #1163
+- Fixed memory leak on long running room/participants #1169
+- Force full reconnect when there is no previous answer #1168
+- Fixed potential SSRC collision between participants #1173
+- Prevent RTX buffer and forwarding path colliding #1174
+- Do not set forceRelay when unset #1184
+- Prevent subscription after participant close #1182
+- Fixed lost RTCP packets when incorrect buffer factory was used #1195
+- Fixed handling of high bitrate while adding Opus RED #1196
+- Fixes a rare timing issue leading to connection failure #1208
+- Fixed incorrect handling of | in participant identity #1220 #1223
```

Comment:
Should we add some note about advising people to upgrade to this and that the old key method will be deprecated in a subsequent release?

Reply:
yeah I'll include it in the GH release notes.